get_instagram.py

```diff
@@ -0,0 +1,27 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+	self.username = username
+	self.password = password
+	self.api = InstagramAPI(username=user_name, password=password)
+	self.api.login()
+	self.user_id = self.api.username_id
+
+    def get_followers(self):
```

Comment:
Please add `user_id` as a parameter.

It might be set as a `None` by default.

Reply:
fine